proxy/http/HttpConfig.h

```diff
@@ -851,8 +852,8 @@ extern volatile int32_t icp_dynamic_enabled;
 /////////////////////////////////////////////////////////////
 inline HttpConfigParams::HttpConfigParams()
   : proxy_hostname(NULL), proxy_hostname_len(0), server_max_connections(0), origin_min_keep_alive_connections(0),
-    max_websocket_connections(-1), disable_ssl_parenting(0), enable_url_expandomatic(0), no_dns_forward_to_parent(0),
-    uncacheable_requests_bypass_parent(1), no_origin_server_dns(0), use_client_target_addr(0), use_client_source_port(0),
+    max_websocket_connections(-1), disable_ssl_parenting(0), enable_url_expandomatic(0),
+    no_dns_forward_to_parent(0), no_origin_server_dns(0), use_client_target_addr(0), use_client_source_port(0),
```

Comment:
Well we made compromises :)

Since the list is so long, can we make a similar change to the one you have don't for array initializers:

```C
no_dns_forward_to_parent(0),
no_origin_server_dns(0),
use_client_target_addr(0),
use_client_source_port(0),
```

Not proposing a general change in standards, just making a special case for this instance since it is so hard to read all the initializers.